### DISCUSSION TITLE
How to use generated Adapter files from lora finetuning? for inference?

### DISCUSSION THREAD

**Discussion by nighting0le01** [haotian-liu/LLaVA#391] [General]
How to use generated Adapter files from lora finetuning? for inference? it says missing KeyError: 'LlavaConfig'
![image](https://github.com/haotian-liu/LLaVA/assets/81643693/16c8cbcf-c9e9-4517-acdf-437eb8232f95)
this is what gets generated. how to convert this to llava weights to use with gradio

**Comment by haotian-liu** [comment]
Please check out the instructions here: https://github.com/haotian-liu/LLaVA/blob/main/docs/LoRA.md
